opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/mocks/WiremockAccountInformationRequest.java

```diff
@@ -139,14 +138,13 @@ public SELF user_anton_brueckner_provided_initial_parameters_to_list_accounts_wi
     }
 
     public SELF user_provided_initial_parameters_in_body_to_list_accounts_with_all_accounts_consent_with_ip_address_check(String body) {
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response = withSignatureHeaders(RestAssured
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests